fossa/pr-scan/scan.sh

```
@@ -0,0 +1,106 @@
+# --- m2 settings (optional; only mounted if present) ---
+m2_settings="${HOME}/.m2/settings.xml"
+
+docker_pull() { docker pull "$1" >/dev/null; }
+
+run_scan() {
+  local scan_img="$1" gdc_conf="$2"
+  local java_version="" conf_arg=()
+  local mount_args=()
+
+  if [ -n "${gdc_conf}" ] && [ -f "${gdc_conf}" ]; then
+    local rel_conf="${gdc_conf#${src_dir}/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)